metabase/metabase
# Fix whitelabelling of CSS colors with the new palette

Epic https://github.com/metabase/metabase/issues/22816

A quick fix for the whitelabel issue with the new palette. A better solution would require migration from all `color-mod` usages in our CSS code.

How to test:
- Run Metabase EE
- Go to `/admin/settings/whitelabel`
- Change the first chart color and reload the page
- Make sure the brand color (e.g. the primary button color) is unchanged

<img width="1373" alt="Screenshot 2022-06-06 at 12 13 15" src="https://user-images.githubusercontent.com/8542534/172132531-b252080e-8b9a-4ba9-890b-bdc632a44bde.png">
